packages/gatsby-theme-carbon/src/components/EditLink/EditLink.module.scss

```
@@ -1,6 +1,11 @@
 .link {
-  color: $text02;
+  color: $text-02;
   &:hover {
-    color: $text01;
+    color: $text-01;
   }
 }
+
+.row {
+  position: relative;
+  bottom: calc(-160px + 0.875rem + 32px);
```

```suggestion
  top: 9rem;
```
We want `calc(160px - 14px) = 144px = 9rem`. Spec is from [this comment](https://github.com/carbon-design-system/carbon-website/issues/862#issuecomment-467622946)